docs/changelog/breaking-changes.stories.mdx

```diff
@@ -7,6 +7,12 @@ import { Meta } from "@storybook/addon-docs";
 
 # Upcoming Breaking Changes
 
+## `@jobber/components@4.91.0`
+
+Moving forward, to import Atlantis Styles you will have to import
+`@jobber/components/styles` or `@jobber/components/dist/styles.css` into your
+project depending on if you're in an ESM or CJS environment respectively.
+
```

Comment:
Is this a change that _everyone_ installing Atlantis for the first time would need to do, or is this only a required step for existing consumers?

Reply:
Also, this might be a designer thing, but linking to definitions of what ESM or CJS are would probably be useful!